src/refinement-list/refinement-list.ts

```diff
@@ -151,4 +152,19 @@ export class NgAisRefinementList extends BaseWidget {
       this.state.refine(item.value);
     }
   }
+
+  public updateState = (state: RefinementListState): void => {
+    this.state = {
+      ...state,
+      items: state.items.map(item =>
+        Object.assign({}, item, {
+          _highlightResult: {
+            item: {
```

Comment:
This doesn't satisfy the type for a highlighted result:

```ts
type HitAttributeHighlightResult = {
  value: string;
  matchLevel: 'none' | 'partial' | 'full';
  matchedWords: string[];
  fullyHighlighted?: boolean;
};
```

Since the engine doesn't return all these properties for SFFV matches, should we fake them?